### PR TITLE
Ns 162/dto for total http requests

### PR DESCRIPTION
**Pull request by net-illia-stetsenko**
Implemented api for total http requests chart

### CODE REVIEW

**Comment by net-illia-stetsenko** [comment]
Also deleted unused dashboard api

File: net-reporter-api/src/api/total_http_requests/total_http_requests_filters.rs [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+            bytes_lower_bound,
+            bytes_upper_bound,
+        }
+    }
+
+    pub fn get_http_methods(&self) -> &[String] {
+        self.http_methods.as_slice()
+    }
+
+    pub fn is_include_http_methods_mode(&self) -> Option<bool> {
```
Comment (net-yehor-tretiakov) [inline net-reporter-api/src/api/total_http_requests/total_http_requests_filters.rs:51]: There might be a conversation about if we should name returning `Option<T>` methods `is_..`

File: net-reporter-api/src/api/total_http_requests/total_http_requests.rs [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+    use net_core_api::encoder_api::Encoder;
+    use net_core_api::decoder_api::Decoder;
+
+    use crate::api::total_http_requests::http_requests_bucket::HttpRequestsBucketDTO;
+    use crate::api::total_http_requests::total_http_requests::TotalHttpRequestsDTO;
+
+
+    #[test]
+    fn reader_correctly_read_encoded_graph_edge() {
```
Comment (net-yehor-tretiakov) [inline net-reporter-api/src/api/total_http_requests/total_http_requests.rs]: `..._graph_edge` COPIUM

File: net-reporter-api/src/api/total_http_requests/total_http_requests_filters.rs [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+    use ion_rs::StreamItem;
+    
+    use net_core_api::encoder_api::Encoder;
+    use net_core_api::decoder_api::Decoder;
+
+    use crate::api::total_http_requests::total_http_requests_filters::TotalHttpRequestsFiltersDTO;
+
+    
+    #[test]
+    fn reader_correctly_read_encoded_nbpe_filters_0() {
```
Comment (net-yehor-tretiakov) [inline net-reporter-api/src/api/total_http_requests/total_http_requests_filters.rs]: I hate adding numbers here (in the names)

File: net-reporter-api/src/api/total_http_requests/http_requests_bucket.rs [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    use ion_rs::ReaderBuilder;
+    use ion_rs::StreamItem;
+
+    use net_core_api::encoder_api::Encoder;
+    use net_core_api::decoder_api::Decoder;
+
+    use crate::api::total_http_requests::http_requests_bucket::HttpRequestsBucketDTO;
+
+    #[test]
+    fn reader_correctly_read_encoded_bandwidth_bucket() {
```
Comment (net-yehor-tretiakov) [inline net-reporter-api/src/api/total_http_requests/http_requests_bucket.rs]: Yeap, no ctrl+C 4sure

**Review by net-yehor-tretiakov — CHANGES_REQUESTED** [review]
Get rig of all the ctrl+C (naming) issues

**Pull request merged**